roles/prometheus/tasks/configure.yml

```diff
@@ -58,6 +60,7 @@
   notify:
     - restart prometheus
   become: true
+  when: prometheus_version is version_compare('3.0.0', '<')
```

Comment:
Should probably be just version and not version_compare, afaik the version_compare name has been deprecated

Reply:
Ahh, that was copy-pasta from other roles. Will update it.